ecommerce-federated/README.md

```diff
@@ -0,0 +1,54 @@
+# Ecommerce Federated
+
+The files in this folder contain products (with query suggestions), articles, and FAQ records as you would see in a federated search experience.
+
+## Live demo
+
+You can see a live demo of this dataset [here](https://codesandbox.io/s/github/algolia/autocomplete/tree/next/examples/two-column-layout).
```

Comment:
I'd suggest we also add a link to the guide(s) which use this collection of dataset, so anyone visiting this page can further explore what they can build using those. 

> and learn how those dataset are put in motion in the "Ecommerce federated guide"